### DISCUSSION TITLE
Should I use 5 FPS even though the camera outputs more?

### DISCUSSION THREAD

**Discussion by felipecrs** [blakeblackshear/frigate#2765] [Ask A Question] [answered]
The documentation is not clear about it. My camera does not allow me to change the frame rate. My camera currently seems to be configured to use:

![image](https://user-images.githubusercontent.com/29582865/152878075-9d9dd08e-5750-4373-a31d-afb1b929eff8.png)

Which I guess is 11 FPS (maybe 12)?

So, my question is, should I configure Frigate to detect with 5 FPS or with 11/12?

**Comment by blakeblackshear** [comment, marked as answer]
You should set it to 5 FPS in detect unless you need a higher frame rate. Ffmpeg will discard the extra frames upstream of frigate.

**Reply by felipecrs** [reply]
Got it. Thank you!
